distributed/preloading.py

```diff
@@ -1,4 +1,4 @@
-import atexit
```

Comment:
+1 I like that we're calling the teardown code explicitly in in `close` methods now instead of using `atexit`

Reply:
Yeah, I figure that if people want to run atexit they can use `atexit` in their preload module itself.  We don't have to mandate that structure.